data/json/items/armor/holster.json

```diff
@@ -321,6 +321,47 @@
       }
     ]
   },
+  {
+    "id": "shoulder_holster",
+    "copy-from": "holster",
+    "type": "ARMOR",
+    "name": { "str": "shoulder holster" },
+    "description": "A comfortable quick draw holster for larger pistols.  One side holsters a gun and the other side holds two magazines.  Activate to holster/draw a gun.",
+    "price_postapoc": 500,
+    "material": [ "leather" ],
+    "pocket_data": [
+      {
+        "magazine_well": "200 ml",
+        "pocket_type": "CONTAINER",
+        "holster": true,
+        "min_item_volume": "100 ml",
+        "max_contains_volume": "650 ml",
+        "max_contains_weight": "2 kg",
+        "max_item_length": "250 mm",
+        "moves": 30
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "max_item_length": "240 mm",
+        "max_contains_volume": "750 ml",
```

Comment:
each of these pockets is currently capable of holding 2 30 round stanag magazines